pangolin/4.3.1-pdata-1.30/Dockerfile

```diff
@@ -0,0 +1,217 @@
+RUN pangolin --use-assignment-cache /pangolin/pangolin/test/test_seqs.fasta
+
+# download B.1.1.7 genome from Utah
+ADD https://raw.githubusercontent.com/StaPH-B/docker-builds/master/tests/SARS-CoV-2/SRR13957123.consensus.fa /test-data/SRR13957123.consensus.fa
+
+# test on a B.1.1.7 genome
+RUN pangolin /test-data/SRR13957123.consensus.fa -o /test-data/SRR13957123-pusher && \
+ column -t -s, /test-data/SRR13957123-pusher/lineage_report.csv
+
+ # install unzip for unzipping zip archive from NCBI
+RUN apt-get update && apt-get install -y --no-install-recommends unzip
+
+# install ncbi datasets tool (pre-compiled binary); place in $PATH
+RUN wget https://ftp.ncbi.nlm.nih.gov/pub/datasets/command-line/LATEST/linux-amd64/datasets && \
+ chmod +x datasets && \
+ mv -v datasets /usr/local/bin
+
+# download assembly for a BA.1 from Florida (https://www.ncbi.nlm.nih.gov/biosample?term=SAMN29506515 and https://www.ncbi.nlm.nih.gov/nuccore/ON924087)
+# run pangolin in usher analysis mode
+RUN datasets download virus genome accession ON924087.1 --filename ON924087.1.zip && \
+ unzip ON924087.1.zip && rm ON924087.1.zip && \
+ mv -v ncbi_dataset/data/genomic.fna ON924087.1.genomic.fna && \
+ rm -vr ncbi_dataset/ README.md && \
+ pangolin ON924087.1.genomic.fna -o ON924087.1-usher && \
+ column -t -s, ON924087.1-usher/lineage_report.csv
+
+# test specific for new lineage, XBB.1.16, introduced in pangolin-data v1.19
+# using this assembly: https://www.ncbi.nlm.nih.gov/nuccore/2440446687
+# biosample here: https://www.ncbi.nlm.nih.gov/biosample?term=SAMN33060589
+# one of the sample included in initial pango-designation here: https://github.com/cov-lineages/pango-designation/issues/1723
+RUN datasets download virus genome accession OQ381818.1 --filename OQ381818.1.zip && \
+ unzip -o OQ381818.1.zip && rm OQ381818.1.zip && \
+ mv -v ncbi_dataset/data/genomic.fna OQ381818.1.genomic.fna && \
+ rm -vr ncbi_dataset/ README.md && \
+ pangolin OQ381818.1.genomic.fna -o OQ381818.1-usher && \
+ column -t -s, OQ381818.1-usher/lineage_report.csv
+
+# testing another XBB.1.16, trying to test scorpio functionality. Want pangolin to NOT assign lineage based on pango hash match.
+# this test runs as expected, uses scorpio to check for constellation of mutations, then assign using PUSHER placement
+RUN datasets download virus genome accession OR177999.1 --filename OR177999.1.zip && \
+unzip -o OR177999.1.zip && rm OR177999.1.zip && \
+mv -v ncbi_dataset/data/genomic.fna OR177999.1.genomic.fna && \
+rm -vr ncbi_dataset/ README.md && \
+pangolin OR177999.1.genomic.fna -o OR177999.1-usher && \
+column -t -s, OR177999.1-usher/lineage_report.csv
+
+ ## test for BA.2.86
+ # virus identified in MI: https://www.ncbi.nlm.nih.gov/nuccore/OR461132.1
+RUN datasets download virus genome accession OR461132.1 --filename OR461132.1.zip && \
+unzip -o OR461132.1.zip && rm OR461132.1.zip && \
+mv -v ncbi_dataset/data/genomic.fna OR461132.1.genomic.fna && \
```

Comment:
I think this list of tests is getting very long, and some of what we WERE testing for, we aren't being tested in this image.

What if we did something like
```
# testing the following lineages:
# XBB.1.16 OR177999.1
# BA.2.86 OR461132.1
# JN.2 (BA.2.86 sublineage) JN.2 is an alias of B.1.1.529.2.86.1.2 OR598183.1 (NY CDC Quest sample)
# Q.1 (BA.2.86.3 sublineage); JQ.1 is an alias of B.1.1.529.2.86.3.1 OR716684.1
# JN.1.22 (BA.2.86.x sublineage; full unaliased lineage is B.1.1.529.2.86.1.1.22) PP189069.1 
# JN.1.48 (BA.2.86.x sublineage; full unaliased lineage is B.1.1.529.2.86.1.1.48) PP218754.1
# and so on...
datasets download virus genome accession OQ381818.1,OR177999.1,OR461132.1,OR598183.1,OR716684.1,PP189069.1,PP218754.1,PP770375.1,PQ073669.1,PQ034842.1 && \
unzip ncbi_dataset.zip && \
pangolin ncbi_dataset/data/genomic.fna && \
column -t -s, lineage_report.csv
````